redfish/provider/data_source_redfish_system_boot.go

```diff
@@ -51,7 +206,7 @@ package provider
 // 		"resource_id": {
 // 			Type:        schema.TypeString,
```

Comment:
Please remove these comments too. The rest is good.

Reply:
done